src/nodes/CatchClause.js

```diff
@@ -6,15 +6,22 @@ const {
 
 const printList = require('./print-list');
 
+const parameters = (node, path, print) =>
```

Comment:
just noticing this right now, do we always use function expressions?

Reply:
I tend to favour this notation. Is there an improvement by using declarations instead?

Reply:
not really, but to me, if it's a function I like to signal that w/ the `function` keyword